<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lb_sensor_disc">
   <property name="geometry">
    <rect>
     <x>402</x>
     <y>25</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/light-gray.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_disc_wall">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>40</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/disc-wall-open.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_disc_wall_state">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>20</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OPEN</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_sensor_reward_left">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>210</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/light-gray.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_sensor_reward_right">
   <property name="geometry">
    <rect>
     <x>591</x>
     <y>210</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/light-gray.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_door_state">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>150</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CLOSED</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_door">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>145</y>
     <width>140</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/door-closed.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_reward_left">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>55</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/reward-closed.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_reward_left_state">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CLOSED</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_reward_right">
   <property name="geometry">
    <rect>
     <x>571</x>
     <y>230</y>
     <width>55</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/reward-closed.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_reward_right_state">
   <property name="geometry">
    <rect>
     <x>571</x>
     <y>270</y>
     <width>55</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>CLOSED</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_com_ports">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>165</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Arduino:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_connection">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/wrong.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_door_open">
   <property name="geometry">
    <rect>
     <x>353</x>
     <y>180</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_door_close">
   <property name="geometry">
    <rect>
     <x>413</x>
     <y>180</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_reward_left_close">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_reward_left_open">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_reward_right_close">
   <property name="geometry">
    <rect>
     <x>601</x>
     <y>300</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_reward_right_open">
   <property name="geometry">
    <rect>
     <x>541</x>
     <y>300</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_disc_wall">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>OPEN</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CLOSE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LARGE</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lb_sensor_door">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/light-gray.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_connect">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_disconnect">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_update">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>22</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: #333;
    border: 2px solid #555;
    border-radius: 11px;
    border-style: outset;
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #888
        );
    padding: 5px;
    }

QPushButton:hover {
    background: qradialgradient(
        cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
        radius: 1.35, stop: 0 #fff, stop: 1 #bbb
        );
    }

QPushButton:pressed {
    border-style: inset;
    background: qradialgradient(
        cx: 0.4, cy: -0.1, fx: 0.4, fy: -0.1,
        radius: 1.35, stop: 0 #fff, stop: 1 #ddd
        );}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/update.png</normaloff>images/update.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Count:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_sensor_disc_count">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Count:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_sensor_reward_left_count">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>180</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Count:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_sensor_door_count">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>180</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Count:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_sensor_reward_right_count">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>180</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lb_box">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>0</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/box.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_reward_left">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reward</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bt_reward_right">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>330</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reward</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_reward_left">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>330</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_reward_right">
   <property name="geometry">
    <rect>
     <x>522</x>
     <y>330</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>20</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Time in ms</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>350</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Time in ms</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>lb_box</zorder>
  <zorder>lb_sensor_disc</zorder>
  <zorder>lb_disc_wall</zorder>
  <zorder>lb_disc_wall_state</zorder>
  <zorder>lb_sensor_reward_left</zorder>
  <zorder>lb_sensor_reward_right</zorder>
  <zorder>lb_door_state</zorder>
  <zorder>lb_door</zorder>
  <zorder>lb_reward_left</zorder>
  <zorder>lb_reward_left_state</zorder>
  <zorder>lb_reward_right</zorder>
  <zorder>lb_reward_right_state</zorder>
  <zorder>cb_com_ports</zorder>
  <zorder>label_6</zorder>
  <zorder>lb_connection</zorder>
  <zorder>bt_door_open</zorder>
  <zorder>bt_door_close</zorder>
  <zorder>bt_reward_left_close</zorder>
  <zorder>bt_reward_left_open</zorder>
  <zorder>bt_reward_right_close</zorder>
  <zorder>bt_reward_right_open</zorder>
  <zorder>cb_disc_wall</zorder>
  <zorder>lb_sensor_door</zorder>
  <zorder>bt_connect</zorder>
  <zorder>bt_disconnect</zorder>
  <zorder>bt_update</zorder>
  <zorder>label_9</zorder>
  <zorder>lb_sensor_disc_count</zorder>
  <zorder>label_11</zorder>
  <zorder>lb_sensor_reward_left_count</zorder>
  <zorder>label_12</zorder>
  <zorder>lb_sensor_door_count</zorder>
  <zorder>label_13</zorder>
  <zorder>lb_sensor_reward_right_count</zorder>
  <zorder>bt_reward_left</zorder>
  <zorder>bt_reward_right</zorder>
  <zorder>txt_reward_left</zorder>
  <zorder>txt_reward_right</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
